src/parseMachine.ts

```diff
@@ -6,6 +6,29 @@ import realmsShim from 'realms-shim';
 
 const realm = realmsShim.makeRootRealm();
 
+/**
+ * Wraps a callback in a function to prevent
+ * accessing 'this'
+ */
```

Comment:
This basically repeats the name of the function - do we need comments like this? 🤔

Reply:
Sure, I can ditch if you like